### ISSUE TITLE
[Question] Workshop Blueprint Inventory/Upgrade Chips

### ISSUE THREAD

**Issue by MagicalAbyss** [tsunamayo/Starship-EVO#3351]
### Short Description of the issue:

When player uploading their blueprints to the Steam Workshop or share locally, the upgrade chips that store inside or install on the blueprint also go with it, which other player can simply download of said blueprint and have the same upgrade chips as the uploader. And I can already see this problem being abuse, such as duplicating resources or loots in the future. How will you adjust this problem?

And will there any methods to remove upgrade chips/resources(in the future) in bulk?

### Steps to reproduce the issue:

- Download any blueprints that contains upgrade chip from the Steam Workshop.

**Comment by tsunamayo** [comment]
Yes true, I will wipe the inventory on export. But for survival / career you wont be able to spawn blueprint for free of course. They will have a cost matching the resources to build it.
For the remove in bulk it will wait the inventory update. Thanks
